### ISSUE TITLE
Site must conform to WCAG and W3C standards for accessibility and mark up validity.

### ISSUE THREAD

**Issue by lexpar** [ELVIS-Project/elvis-database#65]
Piece and Movement List templates fail hard (I've updated Collections and Composer lists so far). The templates I have checked and fixed so far are:

base.html
home.html
about.html
search.html (Has header nesting issues)
piece_create.html
collection_list.html (still has empty heading)
composer_list.html (still has empty heading)
